src/nvim/tui/input.c

```diff
@@ -576,7 +609,34 @@ static void tinput_read_cb(Stream *stream, RBuffer *buf, size_t count_,
       }
     }
   } while (rbuffer_size(input->read_stream.buffer));
+}
+
+static void tinput_read_cb(Stream *stream, RBuffer *buf, size_t count_,
+                           void *data, bool eof)
+{
+  TermInput *input = data;
+
+  if (eof) {
+    loop_schedule_fast(&main_loop, event_create(tinput_done_event, 0));
+    return;
+  }
+
+  handle_raw_buffer(input, false);
   tinput_flush(input, true);
+
+  // An incomplete sequence was found. Leave it in the raw buffer and wait for
+  // the next input.
+  if (rbuffer_size(input->read_stream.buffer)) {
+    int ms = get_key_code_timeout();
+    // If ttimeout is not set, start the timer with a timeout of 0 to process
```

Comment:
typo

Reply:
`ttimeout` is the option name. I changed it to be quoted.